site/profile/data/common.yaml

```diff
@@ -190,17 +190,7 @@ profile::archive::common::packages:
   - "cmake"
   - "gcc-c++"
   - "git"
-
-profile::archive::common::python_pips:
-  # install docker-compose at system level - "docker"
-  "docker-compose":
-    virtualenv: "system"
-    ensure: "1.25.0"
-    pip_provider: "pip3"
-  "cryptography":
-    virtualenv: "system"
-    ensure: "3.3"
-    pip_provider: "pip3"
+  - "docker-compose-plugin"
```

Comment:
The spec for this is currently part of 4e4dfdb97c03f99f750f7bdafb2b3b62ccf42a49 and should be moved to be in the same commit.

Reply:
moved